_source/logzio_collections/_security-sources/cynet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Ship logs from Cynet
+logo:
+  logofile: cynet.png
+  orientation: vertical
+data-source: Cynet
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - firewalls
+order: 1380
+---
+[Cynet](https://www.cynet.com/platform/) is a cybersecurity asset management platform. This topic describes how to send system logs from your Cynet platform to Logz.io.
+
+**Before you begin, you'll need**: 
+
+* An active Cynet license
+* Cynet login credentials 
+* An active account with Logz.io
+* Filebeat 7 installed on your machine
+* Root priveleges on your machines 
+
+<div class="tasklist">
+
+
+##### Configure Cynet to send syslog notifications to a remote Syslog server running Filebeat
+
+1. On your Cynet web interface, go to **Setting > Advanced**.
+![Advanced options](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet1.png)
+2. Select the box beside **Send Audit Records to SIEM**.
+3. Go to **Configuration > SIEM settings** and enable the following configuration:
+![SIEM settings](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet2.png)
+   * **IP** - IP
+   * **Port** - 514
+   * **IP address** - `<ADDRESS-OF-YOUR-FILEBEAT-SERVER>>`, e.g. 100.25.154.13:9000
```

Comment:
@DorisNaaman 
we say to set to port 514 but in the example we say port 9000
Do we need both?

Reply:
@nico-shishkin @Doron-Bargo 
i looked at our old Shipping docs for other security products, and on the:
  **host: "<<ADDRESS-OF-YOUR-FILEBEAT-SERVER>>"** - we always wrote "0.0.0.0:<<some port>>
  
  should we stick with old templates ?